src/current/v23.2/live-migration-service.md

```diff
@@ -0,0 +1,598 @@
+
+## Terminology
+
+- A *live migration* involves maintaining two production databases (a source and target database) and replicating data between them until a final cutover.
+- The *source of truth* is the database that serves reads and writes to the application during a live migration. A cutover switches the source of truth.
+- *Shadowing* is the replication of writes from the source of truth to the target database. The LMS supports multiple [shadowing modes](#shadowing-modes).
+
+## Requirements
+
+- [Kubernetes](https://kubernetes.io/) cluster
+- [Helm](https://helm.sh/docs/intro/install/) package manager for Kubernetes
+
+#### Supported databases
+
+- [PostgreSQL]({% link {{ page.version.version }}/migrate-from-postgres.md %}) (source)
+- [MySQL]({% link {{ page.version.version }}/migrate-from-mysql.md %}) (source)
+- CockroachDB (target)
+
+## Installation
+
+The LMS definitions are located in the [`molt` repository](https://molt.cockroachdb.com/):
+
+- [`molt-lms` Helm chart](https://molt.cockroachdb.com/charts/lms/Chart.yaml)
+- [`molt-lms` values file](https://molt.cockroachdb.com/charts/lms/values.yaml) for [configuration](#configuration)
+
+When you install the `molt-lms` chart, the LMS proxy instances and orchestrator are initialized as Kubernetes pods:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+kubectl get pods
+~~~
+
+~~~
+NAME                                          READY   STATUS      RESTARTS      AGE
+lms-molt-lms-5cbd7c748-bzbgn                  1/1     Running     0             2m30s
+lms-molt-lms-5cbd7c748-g5zqf                  1/1     Running     0             2m34s
+lms-molt-lms-5cbd7c748-lbjjz                  1/1     Running     0             2m3
+lms-molt-lms-orchestrator-596d4b54d8-c4v97    1/1     Running     0             21m
+...
+~~~
+
+You will see `lms-molt-lms` pods that match the configured [number of LMS instances](#lms-instances), along with one `lms-molt-lms-orchestrator` pod.
+
+## Configuration
+
+To configure the LMS, override the values defined in [`values.yaml`](https://molt.cockroachdb.com/charts/lms/values.yaml). For information on setting Helm chart values, see the [Helm documentation](https://helm.sh/docs/helm/helm_upgrade/).
+
+{{site.data.alerts.callout_info}}
+Values that are specified in both `lms` and `orchestrator` must have the identical setting.
+{{site.data.alerts.end}}
+
+This section describes the most important and commonly used values.
+
+#### Source dialect
+
+~~~ yaml
+lms:
+  sourceDialect: ""
+...
+orchestrator:
+  sourceDialect: ""
+~~~
+
+You **must** provide a string value for `sourceDialect`, which specifies the dialect of your source database. Supported dialects are:
+
+- `postgres`: PostgreSQL
+- `mysql`: MySQL
+- `crdb`: CockroachDB
```

Comment:
Just checking - this is case-sensitive?

Reply:
Based on the source code, I think we accept `cockroach` and `crdb` to specify the dialect, since we use the dialect flag. I could be mistaken though. @Jeremyyang920 to clarify:
```
const (
	Cockroach Dialect = iota + 1
	MySQL
	PostgreSQL
	CockroachSource
	CockroachTarget
)

var DialectStringRepresentations = map[Dialect][]string{
	Cockroach:       {"crdb", "cockroach"},
	CockroachSource: {"crdb_src"},
	CockroachTarget: {"crdb_dst"},
	MySQL:           {"mysql"},
	PostgreSQL:      {"pg", "postgres", "postgresql", "libpq", "pgsql", "pq", "psql"},
}
```

Reply:
not case sensitive, we take either.